src/watch.rs

```diff
@@ -157,8 +128,8 @@ impl Watch {
         Ok(())
     }
 
-    fn path_is_interesting(&self, path: &PathBuf, kind: &EventKind) -> bool {
-        path_match(&self.watches, path)
+    fn path_is_interesting(watches: &HashSet<PathBuf>, path: &PathBuf, kind: &EventKind) -> bool {
```

Comment:
The switch from `self` to `Self` is probably not worth it unless you want to either a) get rid of `Watch` completely or b) use `path_is_interesting` without having a `Watch` struct at hand in some other module or tests.

Reply:
Mainly because I didn’t want to move it around for now but still wanted to make it not depend on the object anymore. This is just a sanity thing :)